pkg/local_object_storage/metabase/version.go

```diff
@@ -105,3 +110,50 @@ func migrateFrom2Version(db *DB, tx *bbolt.Tx) error {
 
 	return updateVersion(tx, 3)
 }
+
+func migrateFrom3Version(_ *DB, tx *bbolt.Tx) error {
+	c := tx.Cursor()
+	pref := []byte{metadataPrefix}
+	if k, _ := c.Seek(pref); bytes.HasPrefix(k, pref) {
+		return fmt.Errorf("key with prefix 0x%X detected", pref)
```

Comment:
could we add some info why it is bad for us? (looks like already migrated?)

Reply:
u mean to error context? or just a comment in the code

Reply:
to the test message, yes. to show that we are only planning to have these keys but somehow bd already has it

Reply:
added potential reasons